drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1547,11 +1548,16 @@ function DragAndDropBlock(runtime, element, configuration) {
                     var dx = x - drag_origin.x;
                     var dy = y - drag_origin.y;
 
+                    if (typeof item.max_left === 'undefined') {
```

Comment:
Not sure how much this optimisation gets us since it only in use if an item is used multiple times. I think you can just directly set max_left and max_top above instead of setting them on item.

Reply:
The thing is that this code is executed with every move event. Which happens a lot of times while an item is being moved and we only need to calculate the limits once.
Also, we can't calculate the limits on `onDragStart` because the actual div that's dragged is not created yet. It's rendered when `onDragMove` is fired.